mmv1/products/healthcare/terraform.yaml

```diff
@@ -59,6 +59,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           fhir_store_name: "example-fhir-store"
           pubsub_topic: "fhir-notifications"
           bq_dataset_name: "bq_example_dataset"
+        test_vars_overrides:
+          psaRoles: 'BootstrapPSARoles(t,
```

Comment:
I think we will end up needing this in quite a few places, and I'm a little concerned that this format will trip up other contributors. Is there a way would could simplify?

Two ideas that could shorten the call: 1) we probably shouldn't allow this for any other project, so we could omit `projectID`, and 2) while the `*Options` pattern is nice, I don't see us use it very often, and the more verbose call is less ideal in yaml

Reply:
Thanks for the review!

I've tried to make the calls as terse as possible to fit in the yaml.